persistent-mysql/Database/Persist/MySQL.hs

```diff
@@ -1013,10 +1023,12 @@ showSqlType :: SqlType
             -> String
 showSqlType SqlBlob    Nothing    _     = "BLOB"
 showSqlType SqlBlob    (Just i)   _     = "VARBINARY(" ++ show i ++ ")"
+-- The non-default display width "(1)" is kept for now to avoid differences
+-- between persistent versions running against older MySQL versions (eg 5.7):
 showSqlType SqlBool    _          _     = "TINYINT(1)"
 showSqlType SqlDay     _          _     = "DATE"
 showSqlType SqlDayTime _          _     = "DATETIME"
-showSqlType SqlInt32   _          _     = "INT(11)"
+showSqlType SqlInt32   _          _     = "INT"
```

Comment:
This comment doesn't appear to agree with the code change - can you clarify?

Reply:
It's not a great comment - I'll add a commit to try to improve it!  The point is I'm trying to avoid changing *anything* about what is put into the information schema in MySQL 5.7 and early members of the 8.X series.  In 5.7, `INT` gets a default display width `(11)` anyway, so I felt it would be clearer in the future to omit it here (as was already done for `BIGINT`).  However  `(1)` is not the default for `TINYINT`, and omitting it would make MySQL 5.7 add  `(4)` instead in the information schema.  It doesn't affect the data storage for the column itself, of course, but the different information schema might cause spurious migrations in `persistent`, or be incompatible with some external program the user might depend upon.